src/packages/options/index.html

```diff
@@ -19,7 +19,7 @@
       <ul>
         <li><a class="open_tour">Tour</a></li>
         <li><a class="open_wiki">Wiki</a></li>
-        <li><a class="privacy-policy" href="https://github.com/MikkCZ/pontoon-addon/blob/master/PRIVACY.md">Privacy</a></li>
+        <li><a class="privacy-policy" href="../../packages/privacy-policy/dist/index.html">Privacy</a></li>
```

Comment:
Seems like `../privacy-policy/dist/index.html` would also work here?

Reply:
Done, and I addressed your comment from #148 as well.